packages/hash/frontend/src/theme/components/navigation/MuiDrawerThemeOptions.ts

```diff
@@ -1,7 +1,7 @@
 import { Components, Theme } from "@mui/material";
-import { HEADER_HEIGHT } from "../../../components/layout/PageHeader/PageHeader";
 
 export const SIDEBAR_WIDTH = 260;
+export const HEADER_HEIGHT = 64;
```

Comment:
I wonder if this is the best place to set this. I can't think of a better place though

Reply:
I think this should be in the `PageHeader` component, because that's the component this constant is referencing.